src/styles/blocks/page.scss

```diff
@@ -0,0 +1,13 @@
+* {
+  margin: 0;
+  padding: 0;
```

Comment:
don't reset indents under this selector, it's inefficient

Reply:
Could you please explain why it's inefficient if everything is running and tests pased successfully, or how to do it more efficient?

Reply:
Better to reset it in tags that you need to normalize